install/installer/pkg/config/versions/versions.go

```diff
@@ -33,7 +33,6 @@ type Components struct {
 	InstallationTelemetry Versioned `json:"installationTelemetry"`
 	IntegrationTests      Versioned `json:"integrationTests"`
 	OpenVSXProxy          Versioned `json:"openVSXProxy"`
-	PaymentEndpoint       Versioned `json:"paymentEndpoint"`
```

Comment:
@geropl I think we need to exclude this change as it may block rollouts - it's the version manifest and we still have the payment-endpoint component

Reply:
removed for now

Reply:
Ah, understood. :+1: Let's wait one rollout/day.